<launch>
    <node pkg="gps_enu" type="gps_enu" name="gps_enu" output="screen">
        <!-- 注釋後採用全局配置 -->
        <param name="angle_bias" value="-3.5"/>
        <param name="x_bias" value="1.39"/>
        <param name="y_bias" value="0.005"/>
    </node>
    <!-- <node pkg="rviz" type="rviz" name="rviz" output="screen" args="-d $(find gps_enu)/rviz/gps.rviz"/> -->
</launch>